portfolio/src/main/webapp/index.html

```diff
@@ -110,13 +94,24 @@ <h2>
         <div class="desc">Tokyo, Japan</div>
       </div>
 
-      <p>I would love to travel to these cities (and many more) to experience its culture. I personally believe that my life would not be complete withoug immersing myself in places outside my comfort zone. In addition to my love for traveling, I also enjoy taking photos of people, cities, and nature. Capturing a landscape is one of the most beautiful things. Capturing a busy city is even more captivating. I spent my previous summer in New York City, and some of the photos of the city are some of my favourites.</p>
+      <h3>Why?</h3>
+      <p>I would love to travel to these cities (and many more) to experience its culture. I personally believe that my life would not be complete withoug immersing myself in places that I have never been before. In addition to my love for traveling, I also enjoy taking photos of people, cities, and nature. Capturing a landscape is one of the most beautiful things. Capturing a busy city is even more captivating. I spent my previous summer in New York City, and some of the photos of the city are some of my favourites.</p>
       <hr>
-      <h3>
-        <cursive>My Hidden Talent</cursive>
-      </h3>
+      <h3>My Hidden Talent</h3>
       <p>Although I write with my left hand, I am actually ambidextrous. I do many activities with both hands and can even write with both hands (not very well, but its legible)!</p>
-      <p><cursive>Connect with me on <a href=https://www.linkedin.com/in/sydney-beasley-6b04b8168>LinkedIn</a> and <a href=https://www.github.com/Sydney-18>GitHub</a></cursive></p>  
+      <hr>
+      <h2>Social Media</h2>
+
+      <head>  
```

Comment:
`head` must be outside and before the `body` element - see the examples in https://www.w3schools.com/html/html_intro.asp, but your basic layout is always going to be:

Doctype
html
- head /head
- body /body
/html

Reply:
Would you recommend moving these two lines to the top and within head element?

Reply:
Yes, exactly.